src/main/java/com/xebialabs/deployit/ci/JenkinsDeploymentOptions.java

```diff
@@ -83,6 +84,7 @@ public String getDisplayName() {
             return "DeploymentOptions";
         }
 
+        @RequirePOST
         public ComboBoxModel doFillEnvironmentItems(@QueryParameter(value = "credential") @RelativePath(value = "..") String credential,
```

Comment:
This method has `RequirePOST` but no permission check.

Reply:
added permission check